src/admin/forms/BannerForm.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom';
```

Comment:
HOCs aren't all that fashionable in React land any more as they've mostly been replaced by hooks: https://reactjs.org/docs/hooks-intro.html
I think in this case, the `useHistory` hook is what you're looking for.